cilium-cli/connectivity/check/context.go

```diff
@@ -98,8 +99,8 @@ type ConnectivityTest struct {
 
 	lastFlowTimestamps map[string]time.Time
 
-	nodes              map[string]*corev1.Node
-	controlPlaneNodes  map[string]*corev1.Node
+	nodes              map[string]*slimcorev1.Node
```

Comment:
Makes sense to me.  I do wonder, do we have any idea how big of a difference this makes in terms of memory consumption?  Is this targeting some specific issue where a large cluster was using a lot more memory?

Reply:
I don't have exact measurements for the difference in memory consumption, but this came from a situation where we run the connectivity tests as a kubernetes job and that job started getting OOMKilled in clusters of 3000+ nodes when configured with a memory limit of 512Mi